AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/participant-details/participant-details.component.html

```diff
@@ -70,7 +70,13 @@
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-one-third contact-label">Email</div>
         <div id="participant-{{ participant?.ParticipantId }}-email" class="govuk-grid-column-two-thirds wrap">
-          {{ participant?.Email }}
+          <div *ngIf="!isJudge(participant)">{{ participant?.Email }}</div>
+          <div *ngIf="isJudge(participant) && (getJudgeEmail() !== '' || getJudgeEmail() === undefined || getJudgeEmail() !== null)">
```

Comment:
Checking for an empty string, undefined and null can be done in one op - they're all falsy. Instead of checking that the return is no falsy, I recommend just checking it is truthy, like this

```
isJudge(participant) && getJudgeEmail()
```

I would also suggest making `getJudgeEmail` a getter because (a) it has _get_ in its name and (b) it takes no params

Falsy ref: https://developer.mozilla.org/en-US/docs/Glossary/Falsy